gef-ui/server/types/view-models/amendments/facility-value-view-model.ts

```diff
@@ -0,0 +1,12 @@
+import { ViewModelErrors } from '../view-model-errors';
+
+export type FacilityValueViewModel = {
+  dealId: string;
+  facilityId: string;
+  amendmentId: string;
+  facilityValue?: number;
+  currencySymbol: string;
+  exporterName: string;
+  previousPage: string;
+  errors?: ViewModelErrors | null;
```

Comment:
Do we need `null` if errors is optional?

Reply:
This type exists because `validationErrorHandler` returns `ViewModelErrors | null` and is used in the post controllers. On the get controllers there's no errors & the parameter isn't passed in at all. This is widely used across `gef-ui` (and the principle is across portal and tfm-ui). I think its best to be consistent, but possibly something to look at long term

Reply:
Makes sense, thanks